debian/rules

```diff
@@ -3,5 +3,3 @@
 %:
 	dh $@
```

Comment:
I don't know a thing about debian packaging but I think we should copy https://github.com/elementary/switchboard-plug-pantheon-shell/blob/deb-packaging/debian/rules#L13

```suggestion
	dh $@ --buildsystem=meson
```